### PR TITLE
docs: add profanis to GDE resources

### PR DESCRIPTION
**Pull request by profanis**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -744,6 +744,14 @@
     "bio": "Preston is a JavaScript developer, husband, and father of four from Roy, UT. He also writes blog posts, creates training courses for Thinkster.io, and loves learning more about web development. In his free time he likes to spend time with his family and play or watch basketball (Go Jazz!). ",
     "groups": ["GDE"]
   },
+  "profanis": {
+    "name": "Fanis Prodromou",
+    "picture": "profanis.png",
+    "twitter": "prodromouf",
+    "website": "https://blog.profanis.me",
+    "bio": "Fanis is a Full Stack JavaScript developer, Angular GDE, speaker at meetups, author of the Mastering Angular Reactive Forms, and he constantly creates videos in his YouTube channel to share his knowledge with others. He also teaches Angular in Code.Hub institute and mentor other developers. When he is not behind a keyboard, he spent time with his lovely daughter and enjoys family time with his wife.",
```
Comment (petebacondarwin) [inline aio/content/marketing/contributors.json:752]:
```suggestion
    "bio": "Fanis is a Full Stack JavaScript developer, Angular GDE, speaker at meetups, author of the Mastering Angular Reactive Forms, and he constantly creates videos in his YouTube channel to share his knowledge with others. He also teaches Angular in Code.Hub institute and mentors other developers. When he is not behind a keyboard, he spends time with his lovely daughter and enjoys family time with his wife.",
```

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Thanks @profanis

I checked the github and twitter handles. Both are valid. And the website link is good too.

Please can you fix the image size: 

```
Error: The following pictures exceed maximum size limit of 30kb:
  - /home/circleci/ng/aio/content/images/bios/profanis.png
```

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (profanis) [reply aio/content/marketing/contributors.json]: Thanks for the suggestions @petebacondarwin.
I did the changes and then I `git commit --all --fixup HEAD`. 
If this is not OK, I can do an interactive rebase to one commit and force push it

**Review by petebacondarwin — APPROVED** [review]
Spot on @profanis - thanks.

This looks good to me!

**Review by twerske — APPROVED** [review]
Welcome to GDE!

File: aio/content/marketing/contributors.json [hunk 2]
```diff
@@ -744,6 +744,14 @@
     "bio": "Preston is a JavaScript developer, husband, and father of four from Roy, UT. He also writes blog posts, creates training courses for Thinkster.io, and loves learning more about web development. In his free time he likes to spend time with his family and play or watch basketball (Go Jazz!). ",
     "groups": ["GDE"]
   },
+  "profanis": {
+    "name": "Fanis Prodromou",
+    "picture": "profanis.jpg",
+    "twitter": "prodromouf",
+    "website": "https://blog.profanis.me",
+    "bio": "Fanis is a Full Stack JavaScript developer, Angular GDE, speaker at meetups, author of the Mastering Angular Reactive Forms, and he constantly creates videos in his YouTube channel to share his knowledge with others. He also teaches Angular in Code.Hub institute and mentors other developers. When he is not behind a keyboard, he spends time with his lovely daughter and enjoys family time with his wife.",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json:752]: Not a native speaker, but I believe you don't need an article before a book name:

```diff
-[...] author of the Mastering Angular Reactive Forms [...]
+[...] author of Mastering Angular Reactive Forms [...]
```

...or:

```diff
-[...] author of the Mastering Angular Reactive Forms [...]
+[...] author of the book 'Mastering Angular Reactive Forms' [...]
```

Comment (petebacondarwin) [reply aio/content/marketing/contributors.json:752]: Good spot @gkalpak

Comment (profanis) [reply aio/content/marketing/contributors.json]: thanks a lot for this @gkalpak

Comment (gkalpak) [reply aio/content/marketing/contributors.json]: Any time :smiley:

BTW, git does not recognize fixups of fixups, so all fixup commits must refer to the original (non-fixup) commit.
So, you must change your last commit message:

```diff
-fixup! fixup! docs: add profanis to GDE resources
+fixup! docs: add profanis to GDE resources
```

**Comment by AndrewKushnir** [comment]
@profanis thanks for the fixup commit. The `lint` CI job is failing due to the commit message (it has 2 `fixup`s there). Could you please merge all commits into one and let us know (by leaving a comment here) once it's done? Thank you.

**Comment by profanis** [comment]
> @profanis thanks for the fixup commit. The `lint` CI job is failing due to the commit message (it has 2 `fixup`s there). Could you please merge all commits into one and let us know (by leaving a comment here) once it's done? Thank you.

My apologies for this :(
I blindly followed the instructions of this https://github.com/angular/angular/blob/master/CONTRIBUTING.md#reviewing-a-pull-request and didn't spend a minute reading how the fixup commit works.

Lesson learned :)

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 80a90f0ad73de11af7eb0d3af41ba925392508dc.

**Pull request closed without merging**